tomography_preprocessing/utils/star.py

```diff
@@ -34,6 +37,11 @@ def _extract_single_tilt_series_metadata(
 ) -> TiltSeriesMetadata:
     """Get metadata for a specific tilt-series from a tilt-series data STAR file."""
     star = starfile.read(tilt_series_star_file, always_dict=True)
+    #Check if Tomogram Name provided is actually found in the star file
+    if not star['global']['rlnTomoName'].str.contains(tilt_series_id).any():
```

Comment:
this check technically isn't exactly what we want, it will only check that 'rlnTomoName' is a substring of an entry in the dataframe column -> this is fine though, happy to punt on this!